### PR TITLE
OrioleDB doc page - Issue #4886

### PR DESCRIPTION
**Pull request by Aosmankic-AO**
## Description
Created a doc page for the OrioleDB data integration

### CODE REVIEW

File: docs/data-integrations/OrioleDB.mdx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+| ------------------ | ---------------------------------------------------
+| `user`             |  username asscociated with database
+| `password`         |  password to authenticate your access
+| `host`             |  host to server IP Address or hostname                                         
+| `port`             |  port through which TCPIP connection is to be made                  
+| `database`         |  database name connecting to                          
+| ...................|.................................................... 
+
+#Usage
+#In order to make use of this handler and connect to a OrioleDB server in
```
Comment (martyna-mindsdb) [inline docs/data-integrations/OrioleDB.mdx:23]: Please remove `#` at the beginning of this line.

File: docs/data-integrations/OrioleDB.mdx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+| `user`             |  username asscociated with database
+| `password`         |  password to authenticate your access
+| `host`             |  host to server IP Address or hostname                                         
+| `port`             |  port through which TCPIP connection is to be made                  
+| `database`         |  database name connecting to                          
+| ...................|.................................................... 
+
+#Usage
+#In order to make use of this handler and connect to a OrioleDB server in 
+#MindsDB, the following syntax can be used:
```
Comment (martyna-mindsdb) [inline docs/data-integrations/OrioleDB.mdx:24]: Please remove `#` at the beginning of this line.

File: docs/data-integrations/OrioleDB.mdx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+CREATE DATABASE orioledb_data
```
Comment (martyna-mindsdb) [inline docs/data-integrations/OrioleDB.mdx:26]: Please enclose it in a code block like this:

> \```sql
> CREATE DATABASE orioledb_data
> WITH ENGINE = "orioledb",
> PARAMETERS = { 
>   "user": "root",
>   "password": "root",
>   "host": "hostname",
>   "port": 5432,
>   "database": "postgres"
> };
> \```

File: docs/data-integrations/OrioleDB.mdx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  "user": "root",
+  "password": "root",
+  "host": "hostname",
+  "port": 5432,
+  "database": "postgres"
+}
+
+#Now, you can use this established connection to query your database as follows:
+
+SELECT * FROM orioledb_data.loveU LIMIT 10;
```
Comment (martyna-mindsdb) [inline docs/data-integrations/OrioleDB.mdx:38]: Please enclose it in a code block like this:

> \```sql
> SELECT * FROM orioledb_data.loveU LIMIT 10;
> \```

**Comment by martyna-mindsdb** [comment]
recheckcla

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

**Comment by martyna-mindsdb** [comment]
@Aosmankic-AO 
Please make sure to get assigned to the issue before resolving it.

Please implement the requested changes mentioned in the review comments above.

Also, before we can merge your PR, please sign our CLA following the instructions above. Thanks.

**Comment by Aosmankic-AO** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by martyna-mindsdb** [comment]
recheckcla

**Comment by martyna-mindsdb** [comment]
Related to issue https://github.com/mindsdb/mindsdb/issues/4886

**Pull request merged**